test_scenarios/UC4_Activate Computation Task.md

```diff
@@ -0,0 +1,20 @@
+# Test Scenario
+
+## UC4 Activate CT
+
+### Scenariusz główny
+
+##### 1. __App User__ wybiera opcję wyświetlenia zdefiniowanych CT
+##### 2. System wyświetla listę zdefiniowanych CT
+##### 3. __App User__ wybiera opcję "Activate" przy zdefiniowanym CT
+##### 4. System wyświetla informację o poprawnym uruchomieniu CT
```

Comment:
Powinien wyświetlić parametry wejściowe dla weryfikacji

Reply:
Powinien, ale nie robimy przypadków użycia (__Interact with Computation Task__) ani (__Set Data Stores for Computation Task__). Co najwyżej wyświetlenie bez edycji.